### PR TITLE
fix: Add author field to Event model

### PR DESCRIPTION
**Pull request by AndersonSM**
#### What is the purpose of this pull request?

Add an author field to the Event model.

#### What problem is this solving?

The event will now carry the author despite the operation and the document object.

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by jumc — APPROVED** [review]
Approved, but please fix the lint

File: node/clients/webhook.ts [hunk 1]
```diff
@@ -7,8 +7,11 @@ export default class Webhook extends ExternalClient {
       ...options,
       headers: {
         ...options?.headers,
-        'X-PS2WEBHOOK-API-AppToken': context.settings.token,
-        'X-PS2WEBHOOK-API-AppKey': context.settings.key,
+        ...(context.settings.token &&
```
Comment (beatrizmaselli) [inline node/clients/webhook.ts:10]: Nice catch! I just closed the PR I had open to solve this :)

File: node/events/psv2Webhook.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ export async function psv2Webhook(ctx: EventContext<Clients>) {
     profileId: event.profileId,
     operation: event.operation,
     subject: event.entity,
+    author: event.author,
```
Comment (beatrizmaselli) [inline node/events/psv2Webhook.ts:32]: Why we are duplicating this field instead of just adding it inside meta to the missing operation?

**Review by beatrizmaselli — APPROVED** [review]
LGTM! Thanks ❤️

File: node/events/psv2Webhook.ts (hunk 2 above)
Comment (AndersonSM) [reply node/events/psv2Webhook.ts:32]: Because the meta is inside the document and it's not possible to get the document for the delete operation.

**Pull request merged**